README.md

```diff
@@ -1069,6 +1070,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [m3o.com](https://m3o.com) - A cloud platform for API services development. M3O is a fully managed Micro as a Service offering focusing on Go microservices development in the Cloud. Free tier provides enough to run 5 services and collaborate with others.
   * [pipedream.com](https://pipedream.com) - An integration platform built for developers. Develop any workflow, based on any trigger. Workflows are code, which you can run [for free](https://docs.pipedream.com/pricing/). No server or cloud resources to manage.
   * [pythonanywhere.com](https://www.pythonanywhere.com/) — Cloud Python app hosting. Beginner account is free, 1 Python web application at your-username.pythonanywhere.com domain, 512 MB private file storage, one MySQL database
+  * [Square Cloud](https://squarecloud.app/) - Square Cloud is a project born with the aim of providing hassle-free, user-friendly hosting services without requiring advanced knowledge.
```

Comment:
This one has a redirect loop on its english page

<img width="1373" alt="Screenshot 2023-09-22 at 20 43 16" src="https://github.com/ripienaar/free-for-dev/assets/82342/2da41a35-a0ff-46fa-b926-be983ad4bb48">